common/src/main/java/org/mcaccess/minecraftaccess/MainClass.java

```diff
@@ -147,6 +143,18 @@ public static void clientTickEventsMethod(Minecraft minecraftClient) {
         Keystroke.updateInstances();
     }
 
+    private static void initWorldState(LocalPlayer player) {
+        MainClass.inventoryControls = new InventoryControls();
+        MainClass.biomeIndicator = new BiomeIndicator();
+        MainClass.xpIndicator = new XPIndicator();
+        MainClass.facingDirection = new FacingDirection();
+        MainClass.playerStatus = new PlayerStatus();
+        MainClass.playerWarnings = new PlayerWarnings();
+        MainClass.accessMenu = new AccessMenu();
+        MainClass.fluidDetector = new FluidDetector();
```

Comment:
Qualifying `MainClass` is superfluous here.
```suggestion
        inventoryControls = new InventoryControls();
        biomeIndicator = new BiomeIndicator();
        xpIndicator = new XPIndicator();
        facingDirection = new FacingDirection();
        playerStatus = new PlayerStatus();
        playerWarnings = new PlayerWarnings();
        accessMenu = new AccessMenu();
        fluidDetector = new FluidDetector();
```